src/app/admin/categories/modify-categories/modify-categories.component.ts

```diff
@@ -0,0 +1,158 @@
+import { HttpParams } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { STATUS } from 'src/app/shared/constants/app-properties';
+import { ToasterService, TOAST_COLOR_ENUMS } from 'src/app/shared/services/common/toaster/toaster.service';
+import { CommonRequestService } from 'src/app/shared/services/http/common-request.service';
+import { RequestEnums } from '../../../shared/constants/request-enums';
+import Utils from '../../../shared/services/common/utils';
+import { LoaderService } from 'src/app/shared/services/common/loader/loader.service';
+import { ActivatedRoute, Router } from '@angular/router';
+
+@Component({
+  selector: 'app-modify-categories',
+  templateUrl: './modify-categories.component.html',
+  styleUrls: ['./modify-categories.component.scss'],
+})
+export class ModifyCategoriesComponent implements OnInit {
+  categoryForm: FormGroup;
+  STATUS = STATUS;
+  categoryKey: any;
+  categories: [];
```

Comment:
Why we need categories Array here ? No need to show any dropdown here so why we need categories list ?